src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/CreateRoleResponse.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * Copyright 2017 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Surfnet\ServiceProviderDashboard\Domain\ValueObject;
+
+class CreateRoleResponse
+{
+    public function __construct(
+        public readonly int $id,
+        public readonly string $name,
+        public readonly string $shortName,
+        public readonly string $description,
+        public readonly string $urn
```

Comment:
When all fields are `readonly`, is it not allowed to make the entire class `readonly`? Or is this a 8.3 feature? :thinking:

Reply:
You appearently can't use readonly classes in mocks :see_no_evil: 
This is fixed in phpunit 11.5